### DISCUSSION TITLE
Show live speed visually with bar height or color

### DISCUSSION THREAD

**Discussion by userrand** [monkeytypegame/monkeytype#4521] [Ideas]
Currently, the option to show the current typing speed while typing shows a small number at a corner of the typing area. I find this hard to look at while typing. It might be easier to focus on typing if the speed was shown graphically. Some examples:

- A bar that raises or gauge that fills up as the speed increases
-  A circle that changes color as the speed increases.
-  The entire theme changes color as the speed increases.

**Comment by Miodec** [comment]
Have you tried switching the timer/progress style to "text"?

**Reply by userrand** [reply]
Thank you, no I did not realize that there are settings for the size of the speed indicator.

For those reading the current discussion and that do not know where it is, it is at the beginning of the appearance section in the settings and can also be found by typing "timer" in the quick search.


 I like the bar option that is currently in the settings as it is less distracting and more discrete.
